compss/programming_model/bindings/python/setup.py

```diff
@@ -49,7 +50,7 @@
     "-fstack-protector",
 ]
 
-TARGET_OS = os.environ["TARGET_OS"]
+TARGET_OS = platform.system()
```

Comment:
I wasn't sure if there was a reason for using this environment variable, but if it's legacy code, then maybe the `platform` module is a more suitable option? The `TARGET_OS` is not defined in my env.